R/AAADefs.R

```diff
@@ -19,11 +19,11 @@
 
 ## common definitions for arules
 .types <- function(method = "apriori") {
-    targets     <- c("frequent itemsets", "maximally frequent itemsets", 
+    targets     <- c("frequent itemsets", "maximally frequent itemsets", "generator frequent itemsets", 
```

Comment:
/*Makh2018: Adding "generator frequent itemsets" as a target parameter for apriori function*/